src/main/java/org/broadinstitute/hellbender/cmdline/PicardCommandLineProgramExecutor.java

```diff
@@ -0,0 +1,33 @@
+package org.broadinstitute.hellbender.cmdline;
+
+import org.broadinstitute.hellbender.exceptions.GATKException;
+
+/**
+ * Adapter shim for use within GATK to run Picard tools. Note that this class does not have
+ * it's own CommandLineProgramProperties annotation, and isn't intended to be run from the
+ * command line.
+ */
+public class PicardCommandLineProgramExecutor extends CommandLineProgram {
```

Comment:
I do not like this hack at all...for downstream projects is going to be very difficult to include/exclude picard tools...

Reply:
I think this class is fine -- it's just a simple adapter/shim, which is a pretty standard design pattern when integrating multiple codebases.

Reply:
I agree, but it is impossible to include single Picard tools at the moment in `Main` due to this adapter class...